### PR TITLE
Fix e2e with new yamls and support restic again

### PR DESCRIPTION
**Pull request by kumarkgosa**
Fix e2e with new yamls and support restic again

### CODE REVIEW

**Review by jooseppi-luna — APPROVED** [review]

File: pkg/modules/application_mobility.go [hunk 1]
```diff
@@ -92,10 +92,10 @@ const (
 	InitContainerImage = "<INIT_CONTAINER_IMAGE>"
 	//AccessContents - contents of the object store secret
 	AccessContents = "<CRED_CONTENTS>"
-	//AccessKeyID - contains the aws access key id
-	AccessKeyID = "<KEY_ID>"
-	//AccessKey - contains the aws access key
-	AccessKey = "<KEY>"
+	//AKeyID - contains the aws access key id
```
Comment (abhi16394) [inline pkg/modules/application_mobility.go:95]: This was already changed by @jooseppi-luna 
Please confirm with him and also there were more occurrences of this variable
Verify it

File: tests/e2e/testfiles/csm_application_mobility_test_velero.yaml [hunk 2]
```diff
@@ -76,7 +76,11 @@ spec:
           
           # Name of the backup storage url
           # This field has to be changed to a functional backup storage url 
+<<<<<<< HEAD
```
Comment (abhi16394) [inline tests/e2e/testfiles/csm_application_mobility_test_velero.yaml:79]: remove these merge lines

File: tests/e2e/testfiles/csm_application_mobility_test_velero.yaml [hunk 3]
```diff
@@ -94,8 +98,13 @@ spec:
             name: cloud-creds
             #Specify the object store access credentials to be stored in a secret with key "cloud".
             secretContents: 
+<<<<<<< HEAD
```
Comment (abhi16394) [inline tests/e2e/testfiles/csm_application_mobility_test_velero.yaml:101]: remove these merge lines

File: pkg/modules/application_mobility.go [hunk 4]
```diff
@@ -142,11 +142,10 @@ func VeleroCrdDeploy(ctx context.Context, isDeleting bool, op utils.OperatorConf
 	if err != nil {
 		return err
 	}
-	err = applyDeleteObjects(ctx, ctrlClient, yamlString, isDeleting)
-	if err != nil {
-		return err
+	er := applyDeleteObjects(ctx, ctrlClient, yamlString, isDeleting)
```
[the remark this answers is not in the document]
Comment (kumarkgosa) [reply pkg/modules/application_mobility.go:145]: Oh, dont think I made this, it was from the previous PR, it was "er" from the start and then we went to and fro changing this.

**Review by abhi16394 — CHANGES_REQUESTED** [review]

File: pkg/modules/application_mobility.go (hunk 1 above)
Comment (jooseppi-luna) [reply pkg/modules/application_mobility.go:95]: I would just grep and make sure you get everything. Also should run all unit tests and build an image and run e2e tests after any changes

File: tests/e2e/testfiles/csm_application_mobility_test_velero.yaml (hunk 2 above)
Comment (kumarkgosa) [reply tests/e2e/testfiles/csm_application_mobility_test_velero.yaml]: done, cant believe I missed this..

File: pkg/modules/application_mobility.go (hunk 1 above)
Comment (kumarkgosa) [reply pkg/modules/application_mobility.go:95]: done

**Review by mbasha-dell — APPROVED** [review]

**Review by mbasha-dell — APPROVED** [review]

**Pull request merged**